core/constants.go

```diff
@@ -548,3 +548,6 @@ const MultiplyFactorForScCall = 3
 // MaxRoundsWithoutCommittedStartInEpochBlock defines the maximum rounds to wait for start in epoch block to be committed,
 // before a special action to be applied
 const MaxRoundsWithoutCommittedStartInEpochBlock = 50
+
+// MinMetaTxExtraGasCost is the constant defines for minimum gas value to be sent in meta transaction
```

Comment:
"is the constant defined for" or "is the constant that defines the minimum..."

Reply:
defined for :) done.